concepts/05 UI Components/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,186 @@
+You can change any form, editor, or item properties at runtime. In the code below, we modify the [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly) property's value:
+
+---
+##### jQuery
+
+Use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#optionoptionName_optionValue) method to update a Form property, and the [itemOption(id, option, value)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#itemOptionid_option_value) to update the value of an item property.
+
+    <!-- tab: index.js -->
+    $(function() {
+        const form = $("#form").dxForm({
+            formData: {
+                // ...
+            },
+            colCount: 2,
+            items: ["name", "position", "hireDate", "officeNumber"]
```

Comment:
You don't need all this to showcase `readOnly`. The same applies to this code sample for the other frameworks.